assets/terraform-modules/bare-metal/flatcar-linux/kubernetes/bootkube.tf

```diff
@@ -3,8 +3,8 @@ module "bootkube" {
   source = "../../../bootkube"
 
   cluster_name                    = var.cluster_name
-  api_servers                     = [var.k8s_domain_name]
-  etcd_servers                    = var.controller_domains
+  api_servers                     = [format("%s.%s", var.cluster_name, var.k8s_domain_name)]
+  etcd_servers                    = module.controller[0].etcd_servers
```

Comment:
This 0 bugs me here, even though it's correct code :smile:

Reply:
Maybe we can add a comment here, that each instance of controller module generates the same set of `etcd_servers` ?